notebooks/official/matching_engine/sdk_matching_engine_create_multimodal_embeddings.ipynb

```diff
@@ -0,0 +1,1533 @@
+{
```

Comment:
<p>"Datasets can be large, so it's recommended to load a batch of data at a time into memory using a generator."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2164/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='47'/>